e2e/global/common/kamelet_test.go

```diff
@@ -62,21 +59,39 @@ func TestKameletClasspathLoading(t *testing.T) {
 			Eventually(IntegrationPodPhase(ns, "timer-kamelet-integration"), TestTimeoutLong).Should(Equal(corev1.PodRunning))
 
 			Eventually(IntegrationLogs(ns, "timer-kamelet-integration")).Should(ContainSubstring("important message"))
-
-			// Cleanup
-			Expect(Kamel("delete", "--all", "-n", ns).Execute()).Should(BeNil())
 		})
 
 		// Custom repo
 		t.Run("test custom Kamelet repository", func(t *testing.T) {
-
+			globalTest := os.Getenv("CAMEL_K_FORCE_GLOBAL_TEST") == "true"
```

Comment:
I don't think it's good idea. If we know it doesn't run with a global operator then we should move it to `e2e/namespace`, where each operator is certainly installed per test. Probably `e2e/namespace/intall/cli` is the best place to move it.